dataprofiler/profilers/profile_builder.py

```diff
@@ -504,7 +511,33 @@ def _update_row_statistics(self, data):
         self.hashed_row_dict = dict.fromkeys(
             pd.util.hash_pandas_object(data, index=False), True
         )
-        self.null_in_row_count = data.isnull().any(axis=1).sum()
+
+        # Calculate Null Column Count
+        null_rows = {}
+        null_in_row_count = {}
+        first_col_flag = True
+        for column in self.profile:
+            if "statistics" in self.profile[column].profile and \
+                    "null_types_index" in self.profile[column]\
+                    .profile["statistics"]:
+
+                # Cycle through each column and get all the null indices 
+                null_type_dict = self.profile[column].profile["statistics"]\
+                                 ["null_types_index"]
+                null_row_indices = {item for sublist in null_type_dict.values()
+                                    for item in sublist}
+
+                # Find the common null indices between the columns
+                if first_col_flag:
```

Comment:
this if statement is only needed once. Another way to do this would be to set the initial values to the first column outside of the loop and then loop over all other columns.

Another option would be to set the null_rows to be all rows (perhaps memory inefficient) and null_in_row to an empty set.

just some thoughts, none of the necessarily better